doc/platform_api/CMIS_Diagnostic_Monitoring_Overview_in_SONiC.md

```diff
@@ -0,0 +1,971 @@
+    errored_frames_curr_media_input_lowalarm_change_count{lane_num}      = INTEGER ; Errored frames current low alarm change count for media input
+    errored_frames_curr_media_input_highwarning_change_count{lane_num}   = INTEGER ; Errored frames current high warning change count for media input
+    errored_frames_curr_media_input_lowwarning_change_count{lane_num}    = INTEGER ; Errored frames current low warning change count for media input
+    errored_frames_min_host_input_highalarm_change_count{lane_num}       = INTEGER ; Errored frames minimum high alarm change count for host input
+    errored_frames_min_host_input_lowalarm_change_count{lane_num}        = INTEGER ; Errored frames minimum low alarm change count for host input
+    errored_frames_min_host_input_highwarning_change_count{lane_num}     = INTEGER ; Errored frames minimum high warning change count for host input
+    errored_frames_min_host_input_lowwarning_change_count{lane_num}      = INTEGER ; Errored frames minimum low warning change count for host input
+    errored_frames_max_host_input_highalarm_change_count{lane_num}       = INTEGER ; Errored frames maximum high alarm change count for host input
+    errored_frames_max_host_input_lowalarm_change_count{lane_num}        = INTEGER ; Errored frames maximum low alarm change count for host input
+    errored_frames_max_host_input_highwarning_change_count{lane_num}     = INTEGER ; Errored frames maximum high warning change count for host input
+    errored_frames_max_host_input_lowwarning_change_count{lane_num}      = INTEGER ; Errored frames maximum low warning change count for host input
+    errored_frames_avg_host_input_highalarm_change_count{lane_num}       = INTEGER ; Errored frames average high alarm change count for host input
+    errored_frames_avg_host_input_lowalarm_change_count{lane_num}        = INTEGER ; Errored frames average low alarm change count for host input
+    errored_frames_avg_host_input_highwarning_change_count{lane_num}     = INTEGER ; Errored frames average high warning change count for host input
+    errored_frames_avg_host_input_lowwarning_change_count{lane_num}      = INTEGER ; Errored frames average low warning change count for host input
+    errored_frames_curr_host_input_highalarm_change_count{lane_num}      = INTEGER ; Errored frames current high alarm change count for host input
+    errored_frames_curr_host_input_lowalarm_change_count{lane_num}       = INTEGER ; Errored frames current low alarm change count for host input
+    errored_frames_curr_host_input_highwarning_change_count{lane_num}    = INTEGER ; Errored frames current high warning change count for host input
+    errored_frames_curr_host_input_lowwarning_change_count{lane_num}     = INTEGER ; Errored frames current low warning change count for host input
+```
+
+### 2.11 Transceiver VDM flag time set data
+
+The `TRANSCEIVER_VDM_FLAG_TIME_SET` table stores the flag time set for the VDM flags.
```

Comment:
@mihirpat1 better to have separate table for each?

TRANSCEIVER_VDM_HIGH_ALARM_SET_TIME
TRANSCEIVER_VDM_LOW_ALARM_SET_TIME
TRANSCEIVER_VDM_HWARN_ALARM_SET_TIME
TRANSCEIVER_VDM_LWARN_ALARM_SET_TIME